File: IdeaProjects/MyBatisLearning/mybatis-series/chat05/src/main/java/com/javacode2018/chat05/demo1/model/OrderDetailModel.java
package com.javacode2018.chat05.demo1.model;

import lombok.*;

@Builder
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Setter
@Getter
public class OrderDetailModel {
    private Integer id;
    private Integer orderId;
    private Integer goodsId;
    private Integer num;
    private Double totalPrice;
}

File: Java_Kotlin/MyBatisLearning/mybatis-series/chat05/src/main/java/com/javacode2018/chat05/demo3/model/OrderDetailModel.kt
package com.javacode2018.chat05.demo3.model


class OrderDetailModel {
    private var id: Int? = null
    private var orderId: Int? = null
    private var goodsId: Int? = null
    private var num: Int? = null
    private var totalPrice: Double? = null
    override fun toString(): String {
        return "OrderDetailModel(id=$id, orderId=$orderId, goodsId=$goodsId, num=$num, totalPrice=$totalPrice)"
    }

}